feature/google/src/main/java/com/revenuecat/purchases/google/BillingWrapper.kt

```diff
@@ -250,7 +250,12 @@ class BillingWrapper(
         }
 
         synchronized(this@BillingWrapper) {
-            purchaseContext[googlePurchasingData.productId] = PurchaseContext(
```

Comment:
I think this comment is confusing because it's too short. I would be more clear and say that the stored PurchaseContext is associated with the product we are upgrading from, not the one in the `googlePurchasingData`, which is the new product ID